truncate/table_schema.go

```diff
@@ -123,6 +126,20 @@ func fetchTableSchemas(ctx context.Context, client *spanner.Client, targetTables
 			}
 		}
 
+		switch mode {
+		case fetchModeTarget:
+			if _, ok := targets[tableName]; !ok {
+				return nil
+			}
+		case fetchModeExclude:
+			if _, ok := excludes[tableName]; ok {
+				if typ == deleteActionCascadeDelete {
+					excludeParents[parentTableName] = true
+				}
+				return nil
+			}
+		}
```

Comment:
This switch statement handles processing based on the execution mode. fetchModeExclude records the parent table of exclude table that will be deleted by cascade deletion.